heudiconv/info.py

```diff
@@ -43,7 +43,10 @@
     'extras': [
         'duecredit',  # optional dependency
     ],  # Requires patched version ATM ['dcmstack'],
-    'datalad': ['datalad >=%s' % MIN_DATALAD_VERSION]
+    'datalad': ['datalad >=%s' % MIN_DATALAD_VERSION],
+    'physio': [
+        'bidsphysio.dcm2bids >=1.4.3; python_version>"3.5"',   # if dicoms with physio need to be converted
```

Comment:
```suggestion
        'bidsphysio.dcm2bids >=1.4.3',   # if dicoms with physio need to be converted
```

as 3.5 is already below what we support